src/pages/test-report/__tests__/test-report.analytics.effects.spec.ts

```diff
@@ -1258,4 +1260,42 @@ describe('Test Report Analytics Effects', () => {
     });
   });
 
+  describe('reverseLeftPopoverOpened', () => {
+    it('should call logEvent with the correct parameters', (done) => {
+      // ARRANGE
+      store$.dispatch(new testsActions.StartTestReportPracticeTest(testReportPracticeModeSlot.slotDetail.slotId));
+      // ACT
+      actions$.next(new reverseLeftActions.ReverseLeftPopoverOpened());
+      // ASSERT
+      effects.reverseLeftPopoverOpened$.subscribe((result) => {
+        expect(result instanceof AnalyticRecorded).toBe(true);
+        expect(analyticsProviderMock.logEvent).toHaveBeenCalledTimes(1);
+        expect(analyticsProviderMock.logEvent).toHaveBeenCalledWith(
+          `${AnalyticsEventCategories.PRACTICE_TEST} - ${AnalyticsEventCategories.TEST_REPORT}`,
+          `${AnalyticsEventCategories.PRACTICE_TEST} - ${AnalyticsEvents.REVERSE_LEFT_POPOVER_OPENED}`,
+        );
+        done();
+      });
+    });
+  });
```

Comment:
Can we practice BE pages - thought they were all derived off the base page component not the practiceablebasepage? So perhaps an unneccessary test.